BUILD.gn

```diff
@@ -219,6 +219,7 @@ bundle("main_bundle") {
   deps = [
```

Comment:
Add `data = [ "$target_gen_dir/gn_args.txt" ]`

(if it doesn't work, at least add a TODO to do it)